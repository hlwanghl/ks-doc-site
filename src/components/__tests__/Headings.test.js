import React from 'react'
import renderer from 'react-test-renderer'
import Headings from '../Headings'

describe('Headings', () => {
  it('renders correctly', () => {
    const props = {
      headings: [
        { value: 'title 1', depth: 1 },
        { value: 'title 2', depth: 1 },
        { value: 'sub title 1-1', depth: 2 },
      ],
    }

    const tree = renderer.create(<Headings {...props} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
