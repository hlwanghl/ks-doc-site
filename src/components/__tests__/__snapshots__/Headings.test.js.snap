// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headings renders correctly 1`] = `
<div
  className="sc-bdVaJa fEwSeM"
>
  <h5
    className="sc-bwzfXH kgOpLn"
  />
  <div>
    <h5
      className="sc-htpNat uAtax"
      data-anchor="#title-1"
      onClick={[Function]}
      selected={false}
    >
      title 1
    </h5>
    <h5
      className="sc-htpNat uAtax"
      data-anchor="#title-2"
      onClick={[Function]}
      selected={false}
    >
      title 2
    </h5>
    <h5
      className="sc-htpNat uAtax"
      data-anchor="#sub-title-1-1"
      onClick={[Function]}
      selected={false}
    >
      sub title 1-1
    </h5>
  </div>
</div>
`;
